articles/design/creating-invite-only-applications.md

```diff
@@ -1,118 +1,137 @@
+### Construct email templates
 
-* Go to the [APIs section](${manage_url}/#/apis) of the Dashboard.
-* Select **Auth0 Management API**.
-* Click over to the **Machine to Machine Applications** tab.
-* Find the application you just created, and set its toggle to **Authorized**.
-* Use the **down arrow** to open up the scopes selection area. Select the following scopes: `read:users`, `update:users`, `delete:users`, `create:users`, and `create:user_tickets`.
-* Click **Update**.
+The email invitation needs to be sent with your existing email service provider. The language in the email should align with your use case. Include the link generated from the steps above. The text in the email should explain:
+* The next steps to claiming the user account.
+* The expiration of the link.
+* Steps to generate a new invite if it has expired.
 
-![Authorize Application](/media/articles/invite-only/invite-only-authorize-client.png)
+```js
+{% if user.email_verified == false %}
+            <h1>Invitation to our awesome app</h1>
 
-### Import Users
+            <p>Please verify your email address and set your initial password by clicking the following link:</p>
 
-Every user that exists in ExampleCo should be created in your Auth0 database connection as well. Auth0 offers a [bulk user import functionality](/users/bulk-importing-users-into-auth0) for this purpose.
+            <p><a href="{{ url }}">Confirm my account</a></p>
 
-### Email Verification
+  {% else %}
 
-Once you've created the user in Auth0, send a `POST` request from your app to the [Send an email address verification email](/api/management/v2#!/Jobs/post_verification_email) endpoint that includes:
+            <h1>Password Change Request</h1>
 
-* `user_id`: the Auth0 user ID to send the verification email to.
-* `client_id` (optional): the client ID of your app.
+            <p>You have submitted a password change request. </p>
 
-Be sure to update the `MGMT_API_ACCESS_TOKEN` placeholder value below with your [API Access Token](/api/management/v2/tokens).
+            <p>If it wasn't you please disregard this email and make sure you can still login to your account. If it was you, then to <strong>confirm the password change <a href="{{ url }}">click here</a></strong>.</p>
 
-```har
-{
-	"method": "POST",
-	"url": "https://${account.namespace}/api/v2/tickets/email-verification",
-	"httpVersion": "HTTP/1.1",
-	"cookies": [],
-	"headers": [{
-		"name": "Authorization",
-		"value": "Bearer MGMT_API_ACCESS_TOKEN"
-	}],
-	"queryString": [],
-	"postData": {
-		"mimeType": "application/json",
-		"text": "{ \"user_id\": \"abcd|1234\", \"client_id\": \"xyz789\" }"
-	},
-	"headersSize": -1,
-	"bodySize": -1,
-	"comment": ""
-}
+    {% endif %}
+
+            <p>If you have any issues with your account, please don’t hesitate to contact us at 1-888-AWESOMECO.</p>
+
+            <br>
+            Thanks!
+            <br>
```

Comment:
Kill the email example.  The conditional logic is unique to auth0 emails and would end up being confusing in this case